modulefiles/gfsutils_common.lua

```diff
@@ -5,7 +5,7 @@ Load common modules to build GFS utilities on all machines
 local netcdf_c_ver=os.getenv("netcdf_c_ver") or "4.9.2"
 local netcdf_fortran_ver=os.getenv("netcdf_fortran_ver") or "4.6.1"
 
-local bufr_ver=os.getenv("bufr_ver") or "11.7.0"
+local bufr_ver=os.getenv("bufr_ver") or "12.0.1"
```

Comment:
Version 12.0.1 should not be used.  It has an optimization issue in it that causes the GSI to run significantly longer.  This is why the `gsi-addon` environment was created for spack-stack 1.6.0.

If you need to override the version, then do so in `gfsutils_container.intel.lua` by setting the environmental variable `bufr_ver` before loading `gfsutils_common`.